### ISSUE TITLE
Multiple judgments and sets of court orders

### ISSUE THREAD

**Issue by JimKillock** [openrightsgroup/blocked-org-uk#285]
Some cases have multiple judgments and / or have resulted in 2 or 3 sets of court orders, rather than one.

Add a feature to create a second set of court orders + dates, or a second judgment + citation + date.

**Comment by dantheta** [comment]
This is already partly supported - a URL can be added to multiple court orders.  At the moment, the url groups exist within a court judgment (so these aren't currently shared), but this can be changed if needed.  A URL can belong to multiple groups if it is part of multiple court judgments.  

Are there cases where a single court judgment has resulted in multiple court orders for the same URL/network combination?

**Comment by JimKillock** [comment]
Yes, there are a couple of instances where the judgment has granted temporary court orders, which have been followed by a second judgment and further court orders, in this kind of way:

(1) Case number 1234

(a) Judgment HC20131234 issued, 1/2/2013
(b) Court Orders issued 8/2/2013

(c) New judgment HC20145678 issued, 4/5/2014
(d) New Court Orders issued 11/5/2014

**Comment by dantheta** [comment]
OK, I think those could be represented as two court judgments with the same case reference  (and possibly citation?).

If it makes it easier to manage, I can write in a duplicate button so that judgments (with their attached court orders and urls) can be copied to new records quickly.

**Comment by JimKillock** [comment]
The problem with making duplicates is that the URL sets would also need to attach to both. I think that would lead to confusion.

This isn't urgent as we can just use the "other documents" field if we need to for older court orders.

**Comment by dantheta** [comment]
If we keep the same top-level judgment record, but allow multiple court orders for each network within it, would that be better?  That's quite close to what you were describing at the top, but just for court orders rather than judgments as well.

**Comment by JimKillock** [comment]
That would probably work yes. Then older judgments can be entered in the free text field.

**Comment by edjw** [comment]
What's the status of this?

**Comment by dantheta** [comment]
I've made quite a lot of improvements to the court judgment edit screen.  There's even a bit of flashy JS for adding new court order rows.
